cmd/kmd/mlock_windows.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"github.com/algorand/go-algorand/logging"
+)
+
+func tryMlockall(_ logging.Logger) {
```

Comment:
could you please add an info message here, something like 
log.Infof("running on windows -- mlockall not available")
and add a comment that we should look into solutions such as 
https://github.com/elastic/elasticsearch/pull/9186